docs/trace/tail-based-sampling-span-level/README.md

```diff
@@ -0,0 +1,103 @@
+# Tail Based Sampling at a span level: An Example
+
+This is an example / proof of concept to achieve tail-based sampling at a
+span level using the extensibility mechanisms in OpenTelemetry.NET.
+
+This is a way to achieve a combination of:
+
+- Head-based sampling (based on probabilistic sampling) and
```

Comment:
why is this called probabilistic sampling? We are just respecting parent decision here, right?

Reply:
I was trying to contrast head-based sampling (which is a biased/probabilistic sampling) with tail-based sampling (biased/non-probabilistic sampling) - tried to elaborate a bit to clarify this.